src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,35 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    title,
+    completed,
+    user,
+    id,
+  } = todo;
+
+  return (
+    <article
+      data-id={id}
+      className={classNames(
+        'TodoInfo',
+        {
+          'TodoInfo--completed': completed,
+        },
+      )}
+    >
+      <h2 className="TodoInfo__title">{title}</h2>
+      {user && (
```

Comment:
Code style, empty lines between tags
```suggestion
      <h2 className="TodoInfo__title">{title}</h2>
      
      {user && (
```